R/bundle.R

```diff
@@ -185,6 +186,14 @@ createAppManifest <- function(
     }
   }
 
+  # Validate packageRepositoryResolutionR
+  if (!is.null(packageRepositoryResolutionR)) {
+    match.arg(
+      packageRepositoryResolutionR,
+      c("lax", "strict", "legacy", "lockfile")
```

Comment:
Is there's a reason the default values aren't specified in the signature, the "usual way" when using `match.arg()`? (I would believe it if you told me there was a reason, it's just surprising.)

Reply:
If you put them in the signature then the arg won't default to NULL